mk/Common.mk

```diff
@@ -610,14 +610,15 @@ vcs-unit-test:  vcs-run
 ###############################################################################
 # Build disassembler
 
-DPI_DASM_SRC = $(DPI_DASM_PKG)/dpi_dasm.cxx $(DPI_DASM_PKG)/spike/disasm.cc $(SPIKE_PKG)/disasm/regnames.cc
-DPI_DASM_OUT = $(DPI_DASM_PKG)/libdpi_dasm.so
+DPI_DASM_SRC    = $(DPI_DASM_PKG)/dpi_dasm.cxx $(DPI_DASM_PKG)/spike/disasm.cc $(SPIKE_PKG)/disasm/regnames.cc
+DPI_DASM_ARCH   = $(shell uname)$(shell getconf LONG_BIT)
+DPI_DASM_LIB    = $(DPI_DASM_PKG)/lib/$(DPI_DASM_ARCH)/libdpi_dasm.so
 DPI_DASM_CFLAGS = -shared -fPIC -std=c++11
-DPI_DASM_SVDPI_PATH ?= . -please_set_DPI_DASM_SVDPI_PATH_to_where_svdpi.h_is
-DPI_DASM_INC = -I $(DPI_DASM_PKG) -I $(DPI_DASM_SVDPI_PATH) -I $(SPIKE_PKG)/riscv -I $(SPIKE_PKG)/softfloat
+DPI_DASM_INC    = -I$(DPI_DASM_PKG) -I$(DPI_INCLUDE) -I$(SPIKE_PKG)/riscv -I$(SPIKE_PKG)/softfloat
```

Comment:
I would propose that we do not call the DASM-related variables that reference modified spike-files SPIKE_* but rather something akin to "DASM_SPIKE_*" or "SPIKE_DASM_*" in case other spike files are included in the future. There are differences between files coming directly from the spike repository and the ones we included. (Specifically small changes to remove decoding instructions into pseudo instructions which spike does by default). 

This comment applies to all included simulator-specific make files as well (e.g. clean_spike make targets)

Reply:
Good call.  Will uniquify this particular spike.